DXRCore/DeusEx/Classes/DXRActorsBase.uc

```diff
@@ -606,6 +613,13 @@ function bool Swap(Actor a, Actor b, optional bool retainOrders)
     b.SetPhysics(aphysics);
     if(abase != bbase) b.SetBase(abase);
 
+    if (#var(prefix)DataCube(a) != None) {
+        GlowUp(a);
```

Comment:
there are early returns in between, the GlowOff and GlowUp calls would probably be better inside of `SetActorLocation` instead of here